todoApp/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <title>ToDoリスト</title>
+    <script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <h1>ToDoアプリ</h1>
+    <div id="todoList">
+      <h2>ToDo追加</h2>
+      <input v-model="todo" />
+      <button @click="add(todo)">追加</button>
+      <div v-bind:class="{error: true}" v-if="noInput()">
+        ToDoを入力してください
+      </div>
+
+      <h2>ToDo一覧</h2>
+      <table>
+        <tbody>
+          <tr v-for="todo in todoItems" :key="todo.id">
+            <td v-if="todo.isEdit"><input v-model="todo.text" /></td>
+            <td v-else>{{todo.text}}</td>
```

Comment:
Vue.js のサイトの表記と合わせて、前後にスペースを入れておいたほうがいいと思います。

```suggestion
            <td v-else>{{ todo.text }}</td>
```